### DISCUSSION TITLE
开发计划

### DISCUSSION THREAD

**Discussion by TC999** [TC999/AppDataCleaner#7] [General]
## 当前计划：
- [x] 白名单模块（防止误删，保护重要数据
- [ ] 移动文件夹
- [x] 打开文件夹（一直测试失败，待解决
- [ ] 多国语言支持（暂时不考虑，反正鬼佬也不用（不是
- [x] 优化界面
- [ ] 优化代码
- [x] 拆分`ui.rs`代码（好像这里的代码不完全和ui有关
- [ ] 添加软件图标
- [x] 项目网站(使用 gh-pages 实现)
- [ ] 其他……在此讨论中评论即可

**Comment by TC999** [comment]
目前，文件夹描述初稿测试中，下载体验请前往[103测试版3](https://github.com/TC999/AppDataCleaner/releases/tag/v1.0.3-b3)

**Comment by TC999** [comment]
计划：调用系统自带字体，而不是在程序内嵌入

**Comment by TC999** [comment]
代办：根据文件夹大小排序

**Comment by TC999** [comment]
代办：在软件界面编辑文件夹描述并保存 #29

**Comment by fightx** [comment]
楼主，是不是可以增加个移动并创建软链接的方式，类似这样的功能 rmdir  "AppData\Roaming\CC助手" /s /q
mklink /j "AppData\Roaming\CC助手"  "D:\Temp\AppData\Roaming\CC助手"

**Reply by TC999** [reply]
> 楼主，是不是可以增加个移动并创建软链接的方式，类似这样的功能 rmdir "AppData\Roaming\CC助手" /s /q mklink /j "AppData\Roaming\CC助手" "D:\Temp\AppData\Roaming\CC助手"

软件有相关源码，不过似乎不起作用

**Reply by TC999** [reply]
这是源码 <https://github.com/TC999/AppDataCleaner/blob/main/src/move_module.rs>

**Comment by TC999** [comment]
计划新功能：统计总清理数

**Reply by TC999** [reply]
> 计划新功能：统计总清理数

已成功实现 https://github.com/TC999/AppDataCleaner/releases/tag/v1.0.6-b1
